pkg/sql/planner.go

```diff
@@ -626,6 +627,14 @@ func (p *planner) InternalSQLTxn() descs.Txn {
 	return &p.internalSQLTxn
 }
 
+func (p *planner) regionsProvider() *regions.Provider {
+	if txn := p.InternalSQLTxn(); txn != nil {
+		_ = txn.Regions() // force initialization
```

Comment:
nit, why don't we just `return txn.Regions()` ?

Reply:
It doesn't have the concrete type. We'd need to do a type assertion.